### DISCUSSION TITLE
Epoptes LTSP server and client

### DISCUSSION THREAD

**Discussion by Soylor** [epoptes/epoptes#188] [General]
I got a ltsp epoptes working fine. (Fat with images file created from a chroot installation)

i”ll like that all  ltsp clients can run as alumn pcs except some of then that should act as classroom servers.

can i do some personalizarion or tunning in ltsp.conf to achieve this?

**Comment by alkisg** [comment]
I don't understand you question, explain it again with more words.
What is "alumn PC"?

The access to the epoptes GUI is controlled by the "epoptes" group.

**Comment by Soylor** [comment]
Example:

ltsp image jammy (Creates jammy img for iPXE booting)
ltsp initrd

Now i assign using DHCP, host names.

I have lab00 to lab20 Pcs 
lab00 should access to epoptes tool and server for lab01-lab20 Pcs.

So, in ltsp.conf i need define customization to get lab00 act as an epoptes server and lab01-lab20 act as epoptes clients pointing to lab00 epoptes server.

**Comment by alkisg** [comment]
You don't need any configuration at all.
Just make sure that you're using the PPA both inside and outside your chroot.
This will give you epoptes 23.01.
This will allow anyone in the epoptes group to run epoptes wherever he may be working, either in lab00 or in lab20.

See: https://github.com/epoptes/epoptes/issues/182

**Comment by Soylor** [comment]
Every pc boots with ipxe, so i need some of then execute epoptes-client with server parameter (at boot time) .
If i boot a hostname lab01, it shoud connect to lab00-server
If i boot a hostname work01, it shoud connect to work00-server

So i need to configure:

- If Hostname is `lab01-lab99` it shoud get lab from hostname and connect to `lab00` server.
- if Hostname is `lab00 `(or any `name00`) then it allows any `user00` account logged to execute **epoptes**.

User00 accounts are teache accounts
Pc00 hostnames are classroom teacher PC.

**Comment by alkisg** [comment]
Then you'd use this documentation:
https://epoptes.org/documentation/run-fat/#locally

**Comment by Soylor** [comment]
yes, it 's fine but `[hostname]` sections dows not work.
Exammple:
`[laba-pc*] ` (for matching client PCs laba-pc01 to laba-pc19) 
It does not work but
`[mac address]`
works

**Comment by alkisg** [comment]
We're tracking this in https://github.com/ltsp/ltsp/discussions/791, no need to replicate that discussion here.
We'll solve it there.
